src/converters/logseq/tests/logseq.test.ts

```diff
@@ -176,3 +176,25 @@ test('Date formats', () => {
     expect(f('date2')?.children[0].refs).toEqual([]);
   }
 });
+
+test('Todos', () => {
+  const [file, f] = importLogseqFile('todo.json');
+  expect(file.summary.topLevelNodes).toEqual(1);
+  expect(file.summary.totalNodes).toEqual(8);
+
+  expect(f('child1')?.type).toBe('node');
+  expect(f('child1')?.name).toBe('later task');
+  expect(f('child1')?.todoState).toBe('todo');
+
+  expect(f('child3')?.type).toBe('node');
+  expect(f('child3')?.name).toBe('done task');
```

Comment:
What about child2 (the NOW)?

May as well check all children of 'id=page1', making it into a table-string, and .toEqual-check the entirety of that (or use inline snapshot if that works in this project :P).

Reply:
NOW/DOING (just different terminology styles of the same thing in LogSeq) is not supported yet, but left them in the tests so that it's clear that we know they exist. I can add more comments.

Reply:
They could be added to the tests still to show it is explicit that it's not supported and transformed. :)

But anyway, this all be fine, if you do an iteration with some extra length to the comments making this a bit clearer for those not well-versed in logseq, that's even better. But feel free to ask for merge whenever. :)